tests/test_backend_minio.py

```diff
@@ -0,0 +1,319 @@
+        fp.write(f"secure = {secure}\n")
+    config = audbackend.backend.Minio.get_config(host)
+    assert config["access_key"] == access_key
+    assert config["secret_key"] == secret_key
+    assert config["secure"]
+
+
+@pytest.mark.parametrize(
+    "interface",
+    [(audbackend.backend.Minio, audbackend.interface.Maven)],
+    indirect=True,
+)
+@pytest.mark.parametrize(
+    "file, version, extensions, regex, expected",
+    [
+        (
+            "/file.tar.gz",
+            "1.0.0",
+            [],
+            False,
+            "/file.tar/1.0.0/file.tar-1.0.0.gz",
+        ),
+        (
+            "/file.tar.gz",
+            "1.0.0",
+            ["tar.gz"],
+            False,
+            "/file/1.0.0/file-1.0.0.tar.gz",
+        ),
+        (
+            "/.tar.gz",
+            "1.0.0",
+            ["tar.gz"],
+            False,
+            "/.tar/1.0.0/.tar-1.0.0.gz",
+        ),
+        (
+            "/tar.gz",
+            "1.0.0",
+            ["tar.gz"],
+            False,
+            "/tar/1.0.0/tar-1.0.0.gz",
+        ),
+        (
+            "/.tar.gz",
+            "1.0.0",
+            [],
+            False,
+            "/.tar/1.0.0/.tar-1.0.0.gz",
+        ),
+        (
+            "/.tar",
+            "1.0.0",
+            [],
+            False,
+            "/.tar/1.0.0/.tar-1.0.0",
+        ),
+        (
+            "/tar",
+            "1.0.0",
+            [],
+            False,
+            "/tar/1.0.0/tar-1.0.0",
+        ),
+        # test regex
+        (
+            "/file.0.tar.gz",
+            "1.0.0",
+            [r"\d+.tar.gz"],
+            False,
+            "/file.0.tar/1.0.0/file.0.tar-1.0.0.gz",
+        ),
+        (
+            "/file.0.tar.gz",
+            "1.0.0",
+            [r"\d+.tar.gz"],
+            True,
+            "/file/1.0.0/file-1.0.0.0.tar.gz",
+        ),
+        (
+            "/file.99.tar.gz",
+            "1.0.0",
+            [r"\d+.tar.gz"],
+            True,
+            "/file/1.0.0/file-1.0.0.99.tar.gz",
+        ),
+        (
+            "/file.prediction.99.tar.gz",
+            "1.0.0",
+            [r"prediction.\d+.tar.gz", r"truth.tar.gz"],
+            True,
+            "/file/1.0.0/file-1.0.0.prediction.99.tar.gz",
+        ),
+        (
+            "/file.truth.tar.gz",
+            "1.0.0",
+            [r"prediction.\d+.tar.gz", r"truth.tar.gz"],
+            True,
+            "/file/1.0.0/file-1.0.0.truth.tar.gz",
+        ),
+        (
+            "/file.99.tar.gz",
+            "1.0.0",
+            [r"(\d+.)?tar.gz"],
+            True,
+            "/file/1.0.0/file-1.0.0.99.tar.gz",
+        ),
+        (
+            "/file.tar.gz",
+            "1.0.0",
+            [r"(\d+.)?tar.gz"],
+            True,
+            "/file/1.0.0/file-1.0.0.tar.gz",
+        ),
+    ],
+)
+def test_maven_file_structure(
+    tmpdir, interface, file, version, extensions, regex, expected
+):
```

Comment:
Although the test name is already telling, a single line docstring would be nice.

Reply:
I added a docstring.